umbraco-cloud/product-upgrades/minor-upgrades.md

```diff
@@ -38,7 +38,7 @@ Once you have the site running locally, you should push your changes to the repo
 The upgrade process left off when it was needing three more steps. These three steps now need to be done manually.
 
 1. Complete the installer
-    * To complete the installer, you should visit the site on its URL. it will be like `https://dev-mysite.s1.umbraco.io`. This will show you the installer screen, where you should insert your backoffice credentials and follow the process. It will run through a few steps, and later Umbraco will be updated to the latest version.
+    * To complete the installer, you should visit the site on its URL. it will be like `https://dev-mysite.euwest01.umbraco.io`. This will show you the installer screen, where you should insert your backoffice credentials and follow the process. It will run through a few steps, and later Umbraco will be updated to the latest version.
```

Comment:
Small suggested to update the sentence structure a bit.

```suggestion
    * To complete the installer, you should visit the site: `https://dev-YOURSITE.euwest01.umbraco.io`. This will show you the installer screen, where you should insert your backoffice credentials and follow the process. It will run through a few steps, and later Umbraco will be updated to the latest version.
```